README.md

```diff
@@ -55,8 +55,7 @@ Read our contributing [guide](https://github.com/arshadkazmi42/currency-symbols/
 ## Contributors
 
 Thank you to all the contributors who help in making this project better :raised_hands:
-
-<a href="https://github.com/arshadkazmi42"><img src="https://github.com/arshadkazmi42.png" width="30" /></a>
+<a href="https://github.com/arshadkazmi42"><img src="https://github.com/arshadkazmi42.png" width="30" /></a><a href="https://github.com/DavidLeeR"><img src="https://github.com/DavidLeeR.png" width="30" /></a>
```

Comment:
Can  you add a line break here.

Reply:
Will do!

Reply:
Sorry for the double commit ! I just wanted to format the markdown a little prettier. I should have labelled the commit better.